README.md

```diff
@@ -12,30 +12,46 @@ If the version you are currently using has not been listed, you can try installi
 
 ## Getting Started
 
-- [Installation Instructions](https://github.com/omise/omise-woocommerce/tree/improve-readme#installation-instructions)
-  - [Manually](https://github.com/omise/omise-woocommerce/tree/improve-readme#manually)
-- [First Time Setup](https://github.com/omise/omise-woocommerce/tree/improve-readme#first-time-setup)
-  - [Connect your store with your Omise account](https://github.com/omise/omise-woocommerce/tree/improve-readme#connect-your-store-with-your-omise-account)
-  - [Enable payment methods](https://github.com/omise/omise-woocommerce/tree/improve-readme#enable-payment-methods)
+- [Installation Instructions](#installation-instructions)
+  - [WordPress Plugin Manager (recommended)](#wordpress-plugin-manager-recommended)
+  - [Manually](#manually)
+- [First Time Setup](#first-time-setup)
+  - [Connect your store with your Omise account](#connect-your-store-with-your-omise-account)
+  - [Enable payment methods](#enable-payment-methods)
 
 ...
 
 ### Installation Instructions
 
-In order to install Omise-WooCommerce plugin, you can either manually download the plugin from this repository and install it or download it via WordPress Plugin Store.  
+In order to install Omise-WooCommerce plugin, you can either download it via WordPress Plugin Store or manually download the plugin from this repository.
+
+#### WordPress Plugin Manager (recommended)
+
+You can simply install Omise WooCommerce plugin via WordPress Plugin Manager by the following 4 steps
+1. From the left sidebar at your WordPress Admin page, under **"Plugin"** section. Click **"Add New"**.
+
+2. At the **"Add Plugins"** page, search for Omise WooCommerce plugin using keyword: `Omise`.
+
+3. Click **"Install Now"** to download and install Omise WooCommerce plugin into your WordPress website.
+
+![Screen_Shot_2562-10-01_at_08 28 27_copy](https://user-images.githubusercontent.com/2154669/68250269-274f1080-0053-11ea-8db1-bab9cc32ea46.png)
+
+4. After the plugin has been downloaded and installed. The **"Install Now"** button will now be changed to `Activate`. Make sure to click **"Activate"**
```

Comment:
This one button is expressed in this document three ways: 

```
`Activate`
**"Activate"**
`activate`
```

Of the three, I prefer the middle one (I'm agnostic on the bold).